### PR TITLE
removed unnecessary abstract

### LINKED ISSUE #13: Changes according to PR review of create note listing page ui and create note deletion page ui.

### PR DESCRIPTION
**Pull request by anmol-kumar0815**
Fixes #13 
@yedhink _a
Please review.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/List.jsx [hunk 1]
```diff
@@ -2,11 +2,11 @@ import React from "react";
 
 import Note from "./Note";
 
-const Box = ({ fetchNotes, notes }) => (
+const List = ({ fetchNotes, notes }) => (
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/List.jsx:5]: Do we need to abstract this into a separate component?

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 2]
```diff
@@ -1,15 +1,71 @@
...
+    setSelectedNoteIds([note.id]);
+  };
+
+  return (
+    <Container key={note.id}>
+      <div className="flex justify-between text-xl font-bold">
+        <Typography style="h3">{note.title}</Typography>
+        <Dropdown buttonStyle="text" icon={MenuVertical}>
+          <li>Edit</li>
+          <li onClick={handleDelete}>Delete</li>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:27]: Go through the Slack thread.

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 3]
```diff
@@ -1,15 +1,71 @@
...
+      {showDeleteAlert && (
+        <DeleteAlert
+          refetch={fetchNotes}
+          selectedNote={selectedNote}
+          selectedNoteIds={selectedNoteIds}
+          setSelectedDeleteNote={setSelectedNote}
+          onClose={() => setShowDeleteAlert(false)}
+        />
+      )}
+      <div className="mb-4 flex text-left">
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:39]: Do we need this div? Why do we even need `flex` here?

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 4]
```diff
@@ -1,15 +1,71 @@
...
+        <Typography style="body3">{note.description}</Typography>
+      </div>
+      <div className="flex items-center justify-between">
+        <Button
+          className="mx-2 mt-3"
+          label="Getting Started"
+          style="secondary"
+        />
+        <span className="flex justify-between">
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:48]: Why did you prefer `span` over `div` here?

**Review by yedhink — COMMENTED** [review]
@anmol-kumar0815

File: app/javascript/src/components/Dashboard/Notes/List.jsx (hunk 1 above)
Comment (anmol-kumar0815) [reply app/javascript/src/components/Dashboard/Notes/List.jsx:5]: Fixed this.

File: app/javascript/src/components/Dashboard/Notes/Note.jsx (hunk 3 above)
Comment (anmol-kumar0815) [reply app/javascript/src/components/Dashboard/Notes/Note.jsx:39]: I was trying to make every component inside a separate container, But now I got it, It was unnecessary over here.
Actually, I was experimenting something with flex and forget to remove it after doing that, I am sorry for this.

File: app/javascript/src/components/Dashboard/Notes/Note.jsx (hunk 4 above)
Comment (anmol-kumar0815) [reply app/javascript/src/components/Dashboard/Notes/Note.jsx:48]: I wanted to display Note details in a single line, I thought for inline organization span would be a good choice that’s why I used span.
I will read more about when to use span and div.

**Pull request merged**